LuckyDragon/app/src/main/java/com/example/luckydragon/Models/User.java

```diff
@@ -129,6 +127,23 @@ public void save() {
                 });
     }
 
+    public void deleteUser(String deviceId) {
```

Comment:
get rid of deviceId parameter and use instance attribute for id

also make this remove itself from the userlist if it exists by calling a function on userlist that also notifies observers

Reply:
Done

Reply:
param still there in function